internal/lm/engine.go

```diff
@@ -40,7 +40,7 @@ func NewLMEConnection(data chan []byte, errors chan error, status chan bool) *LM
 func (lme *LMEConnection) Initialize() error {
 	err := lme.Command.Open(true)
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
```

Comment:
need unit tests for this

Reply:
weren't there originally when I started on this story but I'll add them. not sure why they weren't there first?

Reply:
done